adhocracy4/comments_async/api.py

```diff
@@ -153,3 +152,23 @@ def destroy(self, request, *args, **kwargs):
                              instance=comment)
         serializer = self.get_serializer(comment)
         return Response(serializer.data)
+
+    def list(self, request, *args, **kwargs):
+        """Add organisation terms of use info to response.data."""
+        response = super().list(request, args, kwargs)
+        if response.status_code == 400:
+            return response
```

Comment:
Is this for the pagination in combination with linking to a comment? That works still fine, then?

Reply:
@fuzzylogic2000 I added that, because a test was failing. It failed, because for a bad request the response is empty, but I was still trying to add something to it. So I copied that from the PermissionInfoMixin above.. I dont think that changes anything for pagination and comment linking, right? Or what do you mean exactly?